docs/getting-started/cli-getting-started.md

```diff
@@ -60,43 +60,58 @@ See [Understanding core command groups](../user-guide/cli-using-understanding-co
+Zowe CLI V3-LTS supports **team** profiles. The process of setting up team profiles is simple and can be rolled out easily across your organization. We highly recommend that you configure team profiles to support your Zowe CLI implementation. For more information, see [Using team profiles](../user-guide/cli-using-using-team-profiles.md).
+
 ## Using profiles
 
-Zowe profiles let you store configuration details such as username, password, host, and port for a mainframe system. Switch between profiles to quickly target different subsystems and avoid typing connection details on every command.
+Zowe profiles let you store configuration details such as the username, password, host, and port for a mainframe system. Switch between profiles to quickly target different subsystems and avoid typing connection details on every command.
 
 ### Profile types
 
-Most command groups require a `zosmf-profile`, but some plug-ins add their own profile types. For example, the CICS plug-in has a `cics-profile`. The profile type that a command requires is defined in the `PROFILE OPTIONS` section of the help response.
+Most command groups require a `zosmf-profile`, but some plug-ins add their own profile types. For example, the CICS plug-in has a `cics-profile`.
+
+The profile type that a command requires is defined in the `PROFILE OPTIONS` section of the `--help` response.
+
+:::tip
 
-**Tip:** The first `zosmf` profile that you create becomes your default profile. If you don't specify any options on a command, the default profile is used. Issue `zowe profiles -h` to learn about listing profiles and setting defaults.
+The first `zosmf` profile that you create becomes your default profile. If you do not specify any options on a command, the default profile is used. Issue `zowe config list --help` to learn about listing profiles and setting defaults.
 
-### Creating zosmf profiles
+:::
+
+### Creating z/OSMF profiles
+
+Create a global configuration file:
 
 ```
-zowe profiles create zosmf-profile myprofile123 --host my.company.com --port 123 --user myusername123 --password mypassword123
+zowe config init --global-config
 ```
 
-**Notes:** 
+A `zowe.config.json` file is saved in your home `~/.zowe` directory that includes a z/OSMF profile. Use a text editor to add or modify connection information in the profile.
+
+:::note Notes 
 
-- The port defaults to 443 if you omit the `--port` option. Specify a different port if your host system does not use port 443.
-- If z/OSMF is configured for high availability in Sysplex, create the CLI zosmf-profile with DVIPA address/hostname to ensure availability of REST services. For more information, see [Configuring z/OSMF high availability in Sysplex](../user-guide/systemrequirements-zosmf-ha.md).
+- Use the preceding command only when a configuration file does not already exist.
```

Comment:
Should we also mention that the preceding command will also prompt for connection information?
- host
- user
- password

Note, we do not prompt for `port` since we assume the default `443` 
So if people do not have z/osmf on that port, they better change it 😅